mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -644,10 +662,10 @@ message PayAddressCodeRequest {
     bytes sender_monitor_id = 1;
 
     // Subaddress the funds are coming from.
-    uint64 sender_subaddress = 2;
+    uint64 sender_subaddress_index = 2;
 
     // Base-58 encoded "MobileCoin Address Code"
-    string receiver_b58_code = 3;
```

Comment:
I think this might be better unchanged? @eranrund

Reply:
`receiver_b58_code` seems better to me